README.md

```diff
@@ -1,25 +1,115 @@
 # glazier
 
-`glazier` is an attempt to provide a common interface to the various
-elements of different platform application frameworks. It is designed to be used
-by [druid], a UI toolkit.
+Glazier is a portable infrastructure layer intended for high quality GUI
+toolkits in Rust. It is based on druid-shell but with some changes, primarily
+around making it agnostic to the choice of drawing.
+
+Primary platforms are Windows, macOS, and Linux. Web is a secondary platform.
+Other ports may happen if they are contributed.
+
+This library is currently work in progress and should be considered experimental.
+Contributions are welcome, see [CONTRIBUTING](./CONTRIBUTING.md) for more details.
+
+## Scope
+
+The following tasks are in scope. Mostly they are implemented, but as always
+there is more refinement to be done.
+
+* Window creation, including subwindows.
+
+* System menus. These are especially important on macOS.
+
+* Keyboard events.
+
+* Input Method Editor. On macOS, correct handling of dead keys is through the
+IME, and an application built on Glazier is expected to handle IME.
+
+* Mouse and pointer events. Mouse events are currently supported, pointers
```

Comment:
Is touch something that should be in scope?

Reply:
Yes, see above.